gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -74,11 +74,10 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
             # If a channel was explicitly provided, set it.
             self._grpc_channel = channel
         elif api_mtls_endpoint:
-            host = (
-                (":" in api_mtls_endpoint)
-                and api_mtls_endpoint
-                or (api_mtls_endpoint + ":443")
-            )
+            if ":" in api_mtls_endpoint:
+                host = api_mtls_endpoint
+            else:
+                host = api_mtls_endpoint + ":443"
```

Comment:
This is fine as is, but what do you think about
```python
host = api_mtls_endpoint if ":" in api_mtls_endpoint else api_mtls_endpoint + ":443"
```
?
Judgement call on your part.

Reply:
yes, the one line code looks better, applied the change